src/cityreader/cityreader.py

```diff
@@ -21,7 +28,12 @@ def cityreader(cities=[]):
   # Ensure that the lat and lon valuse are all floats
   # For each city record, create a new City instance and add it to the 
   # `cities` list
-    
+    with open("cities.csv", newline='') as city_file:
+        import csv
```

Comment:
It's _usually_ considered un-Pythonic to add imports anywhere other than the top of the Python module unless you have a really good reason. 

Even then, the Zen of Python reminds us: "Special cases aren't special enough to break the rules."